docs/unique/ai-first-os-model-context-protocol.md

```
@@ -64,17 +64,17 @@ We are making an AI first operating system.
 What tools would that have?
 <!-- GENERATED-SECTIONS:DO-NOT-EDIT-BELOW -->
 ## Related content
-- [AI-Centric OS with MCP Layer](ai-centric-os-with-mcp-layer.md)
-- [balanced-bst](balanced-bst.md)
-- [Board Automation Improvements](board-automation-improvements.md)
+- [[ai-centric-os-with-mcp-layer|AI-Centric OS with MCP Layer]]
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)